e2e/fixtures/dotnet/c/project.vcxproj

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <!-- @OctoLinkerResolve(<root>/dotnet/c/foo.cpp) -->
+    <ClCompile Include="foo.cpp" />
+
+    <!-- @OctoLinkerDoesNotResolve -->
+    <ClCompile Include="$(OutDir)strings.cpp">
```

Comment:
@stefanbuck this test failed because it didn't resolve the dependency, and when that happens OctoLinker still injects an `<a>` but doesn't set the `href` attribute. How do we want to handle that scenario?

This is what the html looks like for this line:

```html
<td class="blob-code blob-code-addition">
  <span class="blob-code-inner blob-code-marker" data-code-marker="+">
    &lt;
    <span class="pl-ent">ClCompile</span>
    <span class="pl-e">Include</span>
    =
    <span class="pl-s">
      <span class="pl-pds">"</span>
      <a data-pjax="true" class="octolinker-link">$(OutDir)strings.cpp</a>
      <span class="pl-pds">"</span>
    </span>
    &gt;
  </span>
</td>
```

Reply:
The false positives are resolved in #1069. Will rebase & finish up once that fix gets merged.